tracer/test/snapshots/AwsSnsTests.NetCore.SchemaV0.verified.txt

```diff
@@ -67,13 +67,15 @@
       aws.service: SNS,
       aws.topic.arn: arn:aws:sns:us-east-1:000000000000:MyTopic,
       aws.topic.name: MyTopic,
+      aws_service: SNS,
```

Comment:
Not always available, since user can change this to be from a `ServiceURL`

https://docs.aws.amazon.com/sdkfornet1/latest/apidocs/html/T_Amazon_SimpleNotificationService_AmazonSimpleNotificationServiceConfig.htm